src/Microsoft.DotNet.Wpf/src/Shared/MS/Internal/Text/InternalEncoding.cs

```diff
@@ -0,0 +1,33 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Text;
+
+namespace MS.Internal.Text
+{
+    // We use encodings that are not provided by default in core.
+    // This class makes sure that we register extra providers that are required before use.

```

Comment:
Could you change it to document comment?

    /// <summary>
    /// The comment
    /// </summary>